app/celery/tasks.py

```diff
@@ -482,3 +494,44 @@ def send_inbound_sms_to_service(self, inbound_sms_id, service_id):
                                service_id, inbound_api.url, e))
             except self.MaxRetriesExceededError:
                 current_app.logger.exception('Retry: send_inbound_sms_to_service has retried the max number of times')
+
+
+@notify_celery.task(name='process-incomplete-jobs')
+@statsd(namespace="tasks")
+def process_incomplete_jobs(job_ids):
+    current_app.logger.info("Resuming Job(s) {}".format(job_ids))
+    for job_id in job_ids:
+        process_incomplete_job(job_id)
+
+
+def process_incomplete_job(job_id):
+
+    job = Job.query.filter(Job.id == job_id).one()
+
+    last_notification_added = Notification.query.filter(
+        Notification.job_id == job_id
```

Comment:
Is this query as efficient as getting the max row number?

Reply:
The same? Both are aggregate functions

max might read a little better if there was a .max method but I don't think there is db.func.max(Notification.job_row_number) looked clumsy?